### DISCUSSION TITLE
The app is way too heavy.

### DISCUSSION THREAD

**Discussion by BobSquarePants** [tom-james-watson/breaktimer-app#173] [General]
@tom-james-watson ,

First of all great job about the features of this app! like : 

- Reset on idle (that really work)
- restart break period
- snooze limit

on the other hand too bad that this app is so heavy (windows, Linux I didn't tried)
70MB of install file :scream: 

Multi-process running ( 6 on my system, totaling >40MB of mem ! )

I hope that we will succeed in reducing is heaviness.

Cheers,

**Comment by tom-james-watson** [comment]
This is the reality of electron as it stands. There's not much that can be done about this short of rewriting the whole app, which is not something I personally will be doing.

**Comment by trymeouteh** [comment]
I would like to suggest looking into tauri, tauri is a lightweight version of electron

**Reply by tom-james-watson** [reply]
I'm well aware of electron. If somebody wants to provide a pr rewriting with tauri then please feel free!
